.github/workflows/nightly.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         api-level: [21, 22, 23, 24, 25, 27, 28, 30]
-        target: [google_apis]
+        target: [default]
```

Comment:
@MohitMaliDeveloper I still think `google_apis` are the best. If you want it then we can change it to default and see how nightly goes.
cc @kelson42

Reply:
hi @gouri-panda actually by setting the target is not actualising running the ```google_apis``` while running the emulator , you can check https://github.com/kiwix/kiwix-android/actions/runs/3178395185/jobs/5179833538#step:5:26. So its was showing as on github as they are running ```google_apis``` but eventually its was ```default ``` one. ```google_apis``` are not stable they even fail for android 30 while you can see nightly works well with ```default```.

Reply:
Ok. Let's give them a try.